src/suricata.c

```diff
@@ -2161,8 +2161,10 @@ static int LoadSignatures(DetectEngineCtx *de_ctx, SCInstance *suri)
 {
     if (SigLoadSignatures(de_ctx, suri->sig_file, suri->sig_file_exclusive) < 0) {
         SCLogError(SC_ERR_NO_RULES_LOADED, "Loading signatures failed.");
+#ifndef FUZZ
```

Comment:
Can't the fuzz target simply set `de_ctx->failure_fatal`. I want to avoid such ifdefs whenever possible.

Reply:
Think all you'd need to do is `(void)ConfSetFinal("engine.init-failure-fatal", "0");`

Reply:
Though I'm surprised this is enabled by default? Or did you add it to the target explicitly somehow?

Reply:
Well seen, I was just afraid so I put this ifdef without testing first if it worked without it, testing now